test_runner/regress/test_tenants.py

```diff
@@ -397,3 +398,30 @@ def test_pageserver_with_empty_tenants(
     assert (
         tenant_broken_count == 1
     ), f"Tenant {tenant_without_timelines_dir} should have metric as broken"
+
+
+# Check that empty tenants work with or without the remote storage
+@pytest.mark.parametrize(
+    "remote_storage_kind", available_remote_storages() + [RemoteStorageKind.NOOP]
+)
+def test_pageserver_create_tenants_fail(
+    neon_env_builder: NeonEnvBuilder, remote_storage_kind: RemoteStorageKind
+):
```

Comment:
There have been arguments about this, but I prefer the test comment as python doc comment:



```suggestion
@pytest.mark.parametrize(
    "remote_storage_kind", available_remote_storages() + [RemoteStorageKind.NOOP]
)
def test_pageserver_create_tenants_fail(
    neon_env_builder: NeonEnvBuilder, remote_storage_kind: RemoteStorageKind
):
    """Tenants which fail to be created are cleaned up from disk and left broken"""
```

Then it will be reproduced to the allure report and people don't have to guess what the test is supposed to be doing.

A better name here could be `test_failed_tenant_directory_is_removed` perhaps? Note it needs to be copypasted to the `neon_env_builder.enable_remote_storage` call.

In the descrption I suggested, I saw there was setting the tenant broken. Should we test it with `client.tenant_status(tenant_id)`?